src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -2650,9 +2651,7 @@ impl<'a> TypeChecker<'a> {
             Literal::String(string) => Scalar::String(string.as_bytes().to_vec()),
             Literal::Boolean(boolean) => Scalar::Boolean(*boolean),
             Literal::Null => Scalar::Null,
-            Literal::CurrentTimestamp => Err(ErrorCode::SemanticError(format!(
-                "Unsupported literal value: {literal}"
-            )))?,
+            Literal::CurrentTimestamp => Scalar::Timestamp(Local::now().timestamp_micros()),
```

Comment:
```suggestion
            Literal::CurrentTimestamp => self.resolve_function(*span, "CURRENT_TIMESTAMP", vec![], vec![])
                        .await
```

Reply:
`resolve_function` is an async function.

Reply:
`resolve_scalar_function_call` could change to be sync function.